mobile-app/app/screens/AppNavigator/screens/Portfolio/components/DFIBalanceCard.tsx

```diff
@@ -1,308 +1,111 @@
+    <ThemedViewV2
+      light={tailwind('bg-mono-light-v2-00')}
+      dark={tailwind('bg-mono-dark-v2-00')}
+      style={tailwind('mx-5 mt-2 rounded-lg-v2')}
       testID='dfi_balance_card'
     >
-      <View style={tailwind('flex-col flex-1')}>
-        <ImageBackground
-          source={isLight ? DFIBackground : DFIBackgroundDark}
-          style={tailwind('flex-1 rounded-lg overflow-hidden')}
-          resizeMode='cover'
-          resizeMethod='scale'
-        >
-          <ThemedTouchableOpacity
-            onPress={() => navigation.navigate({
-              name: 'Balance',
-              params: { token: DFIUnified, usdAmount },
-              merge: true
-            })}
-            style={tailwind('flex-1')}
-            dark={tailwind('border-0')}
-            light={tailwind('border-0')}
-            testID='dfi_balance_card_touchable'
-          >
-            <View style={tailwind('flex-row m-4 mb-2 flex-1 justify-between')}>
-              <View style={tailwind('flex-row items-center')}>
-                <DFIIcon width={32} height={32} />
-                <TokenNameText displaySymbol='DFI' name='DeFiChain' testID='total_dfi_label' />
-              </View>
-              {
-                hasFetchedToken
-                  ? (
-                    <TokenAmountText
-                      tokenAmount={DFIUnified.amount}
-                      usdAmount={usdAmount}
-                      testID='dfi_total_balance'
-                      isBalancesDisplayed={isBalancesDisplayed}
-                      denominationCurrency={denominationCurrency}
-                    />
-                  )
-                  : (
-                    <View style={tailwind('pt-1')}>
-                      <View style={tailwind('mb-1.5')}>
-                        <TextSkeletonLoader
-                          iContentLoaderProps={{
-                            width: '150',
-                            height: '16',
-                            testID: 'dfi_balance_skeleton_loader'
-                          }}
-                          textHorizontalOffset='30'
-                          textWidth='120'
-                        />
-                      </View>
-                      <View>
-                        <TextSkeletonLoader
-                          iContentLoaderProps={{
-                            width: '150',
-                            height: '12',
-                            testID: 'dfi_USD_balance_skeleton_loader'
-                          }}
-                          textHorizontalOffset='30'
-                          textWidth='120'
-                        />
-                      </View>
-                    </View>
-                  )
-              }
-            </View>
-          </ThemedTouchableOpacity>
-          {hasFetchedToken && !new BigNumber(DFIUtxo.amount ?? 0).plus(DFIToken.amount ?? 0).gt(0)
+      <View style={tailwind('flex-col flex-1 rounded-lg-v2 overflow-hidden')}>
```

Comment:
removing `flex-1` to resolve card flickering
```suggestion
      <View style={tailwind('flex-col rounded-lg-v2 overflow-hidden')}>
```